torchci/scripts/compute_correlation.py

```diff
@@ -52,6 +52,13 @@ def compute():
         query_lambda="correlation_matrix",
         version=prod_versions["metrics"]["correlation_matrix"],
         workspace="metrics",
+        parameters=[
```

Comment:
I computed correlation for `windows-binary-libtorch` workflow and accidentally added them here, but I guess it's ok to regenerate the correlation matrix anyway.